rb/lib/selenium/webdriver/chrome/options.rb

```diff
@@ -21,12 +21,23 @@ module Selenium
   module WebDriver
     module Chrome
       class Options < WebDriver::Common::Options
-        attr_reader :args, :prefs, :options, :emulation, :extensions, :encoded_extensions
-        attr_accessor :binary
 
         KEY = 'goog:chromeOptions'
 
-        #
+        # see: http://chromedriver.chromium.org/capabilities
+        CAPABILITIES = %i[args binary extensions local_state prefs detach debugger_address exclude_switches
+                          minidump_path mobile_emulation perf_logging_prefs window_types].freeze
+
+        (CAPABILITIES + %i[options emulation encoded_extensions]).each do |key|
+          define_method key do
```

Comment:
How does this look in docs? Should we maybe use macro for better documentation?

Reply:
What I'm trying to say is that these methods are completely hidden in docs right now. YARD is not able to understand them since they are generated dynamically. I was wondering if using [macros](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#macro) could make them visible in docs, but I was wrong.